src/Raven.Server/Documents/PeriodicBackup/Aws/RavenAwsS3Client.cs

```diff
@@ -500,6 +505,101 @@ public async Task DeleteBucket()
             }
         }
 
+        public async Task<IEnumerable<FileInfoDetails>> ListObjects(string prefix, string delimiter, bool listFolders, string continuationToken = null)
+        {
+            var url = $"{GetUrl()}/?list-type=2";
+            if (prefix != null)
+                url += $"&prefix={prefix}";
```

Comment:
check if we need to escape prefix?

Reply:
Does not matter, I will add escape prefix just for safety.

Reply:
ok but please check if it still works after you escape.